deploy/env-uzh-prd/values.yaml

```diff
@@ -0,0 +1,528 @@
+dfDomain: prd.df-app.ch
+
+heartbeats:
+  dailyGroupScores: https://betteruptime.com/api/v1/heartbeat/xN8ytcTixyVXsWnNjk9ndimD
+
+global:
+  appOrigins:
+    api: 'https://backend-sls.klicker.prd.df-app.ch'
+    auth: 'https://auth.klicker.prd.df-app.ch'
+    lti: 'https://lti.klicker.prd.df-app.ch'
+    pwa: 'https://pwa.klicker.prd.df-app.ch'
+    manage: 'https://manage.klicker.prd.df-app.ch'
+    control: 'https://control.klicker.prd.df-app.ch'
+    assessmentApi: 'https://assessment-api.klicker.prd.df-app.ch'
+    assessmentPwa: 'https://assessment.klicker.prd.df-app.ch'
+
+hatchet:
```

Comment:
where do we define the node pool assignments and affinities of the services? e.g., so assessment always runs on assessment, and the other never runs on assessment? in current prod, we at least have affinities